packages/driver/src/cy/chai.ts

```diff
@@ -108,17 +93,19 @@ chai.use((chai, u) => {
     try {
       val && val.document
       val && val.inspect
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
For the errors casted as any:

 Should there be any additional type safety around this? Maybe casting to `unknown` and checking if `isError` is true to handle, otherwise fall through/rethrow or whatever is needed? But that also might be overkill 😅

Reply:
They always throw `Error` and it seems that it is an overkill.

Reply:
Had a feeling 😂 . Works for me!